src/server.c

```diff
@@ -5681,12 +5681,12 @@ int changeBindAddr(sds *addrlist, int addrlist_len) {
         /* Re-Listen TCP and TLS */
         server.ipfd.count = 0;
         if (server.port != 0 && listenToPort(server.port, &server.ipfd) != C_OK) {
-            serverPanic("Failed to restore old listening sockets.");
+            serverPanic("Failed to restore old listening TCP socket.");
```

Comment:
what was wrong with the old text?
is it the fact users can distinguish between this one and the next one?

Reply:
There is nothing wrong with the old text. 
`is it the fact users can distinguish between this one and the next one?`  yes..